test/functionalTest/cases/0970_create_entity/errors_at_creation.test

```diff
@@ -151,27 +151,27 @@ echo
 01. POST /v2/entities without Entity::id
 ========================================
 HTTP/1.1 400 Bad Request
-Content-Length: 61
+Content-Length: 83
 Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
 {
-    "description": "no entity id specified", 
+    "description": "entity id length: 0, min length supported: 1", 
```

Comment:
Not that it is **really** important but here we lose some information.
Non-present and present-but-empty now give the same error description.